### PR TITLE
Remove asyncio loop from start template

### PR DESCRIPTION
**Pull request by bruziev**

### CODE REVIEW

**Comment by asvetlov** [comment]
To avoid confusing better to convert `create_app()` into async function first.

**Comment by samuelcolvin** [comment]
LGTM

File: tests/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 coverage==4.5.1
 docutils==0.14
-flake8==3.5.0
+git+https://github.com/PyCQA/flake8 # TODO: Remove after release https://gitlab.com/pycqa/flake8/merge_requests/230
```
Comment (samuelcolvin) [inline tests/requirements.txt]: probably better to revert `pycodestyle==2.3.1` or whatever rather than use an unreleased version of flake8?

Comment (bruziev) [reply tests/requirements.txt]: Done

**Comment by samuelcolvin** [comment]
Looks good, I'll merge and do a new release next week.

Poke me if I forget.

**Comment by bruziev** [comment]
@samuelcolvin ping

**Pull request merged**